mvapi/settings.py

```diff
@@ -37,7 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-
+    'django_extensions',
```

Comment:
what the purpose of this extension?

Reply:
I use it for having a django shell to develop spikes. It is the only way to launch:
`./manage.py --shell_plus notebook`